plugins/proxies/lib_date.eclmod

```diff
@@ -0,0 +1,91 @@
+/*##############################################################################
```

Comment:
This file does not relate to one of the changes from LN.  Was it released by mistake/

Reply:
I saw this in current <LN>/plugins/ directory. Thought it should be copied to HPCC-Platform. If not I can remove it from HPCC-Platform

Reply:
@richardkchapman not sure what to do about this one.  I don't think we want this plugin included in the system - it is really a legacy LN module, superseded by Std.Date etc.

Should we include all the LN plugins in a separate directory (e.g., plugins/eeproxies)?

It should be deleted from LN if it is being moved to this repo.